ghga_transpiler/core/core.py

```diff
@@ -14,4 +14,41 @@
 # limitations under the License.
 #
 
-"Module including functions to transpile a worksheet"
+"""This module contains functionalities for processing excel sheets into json object."""
+from typing import Union
+
+from openpyxl import Workbook, load_workbook
+
+
+def read_workbook(filename: str) -> Workbook:
+    """
+    Function to read-in a workbook (aka spreadsheet)
+    """
+    return load_workbook(filename)
+
+
+def get_worksheet_rows(
+    worksheet,
+    min_row: Union[int, None],
+    max_row: int,
+    min_col: Union[int, None],
+    max_col: Union[int, None],
```

Comment:
```suggestion
    min_row: Optional[int],
    max_row: int,
    min_col: Optional[int],
    max_col: Optional[int],
```
But when is this actually ever `None`?